### DISCUSSION TITLE
Folia Support across BG plugins?

### DISCUSSION THREAD

**Discussion by VeeShep** [BG-Software-LLC/WildChests#233] [Pending Feature Requests] [answered]
PaperMC's multithreaded server [folia](https://papermc.io/software/folia)
I've checked for any issues made and for any discussions involving the topic and it seems no one has brought the idea up.

**Comment by OmerBenGera** [comment, marked as answer]
Added support to all plugins except SuperiorSkyblock and WildStacker.

WildStacker needs a bit of work for it to work (which I work on), SuperiorSkyblock2 needs API features that are currently lack in Folia (won't be added anytime soon).
